src/request.ts

```diff
@@ -763,7 +783,10 @@ class DatastoreRequest {
       let entities: Entity[] = [];
 
       if (resp.batch.entityResults) {
-        entities = entity.formatArray(resp.batch.entityResults);
+        entities = entity.formatArray(
```

Comment:
Maybe here would be a good place to really friendlify the error message, so that if any error was thrown, it could say “One of the values was out of bounds. To avoid this...” etc.

Reply:
did you mean friendlify this message?
https://github.com/googleapis/nodejs-datastore/pull/516/files/a2eb3e3f1f3ff8c222cc1e8e078932db56886c5e#diff-f6515d62cc6523369af544f6e8852d1cR447

Reply:
Yes! If we can put an error in the right place (like when calling get()), the user would get a message that explains that we tried to return all of the results, but the value for "phoneNumber" was out of bounds. It could also explain to use `wrapNumbers` to circumvent the error.

Reply:
Is this done? This message seems pretty friendly. https://github.com/googleapis/nodejs-datastore/pull/516/files/c4d944e6503221f685bb6499e34554e0107d0d3f#diff-b6ce7cf07299e63c027f59d103d82e81R109